src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/CachePools/config.yml

```diff
@@ -24,8 +24,3 @@ framework:
             cache.pool7:
                 adapter: cache.pool4
                 public: true
-
-services:
```

Comment:
This file seems unused (don't know since when), which probably misled to this change in the original PR.
The file can probably be removed in 3.4.

Reply:
NVM, it's used by `\Symfony\Bundle\FrameworkBundle\Tests\Functional\CachePoolsTest::testCachePools` in the Travis builds.